mcp-server-gitparser/README.md

```
@@ -0,0 +1,88 @@
+docker build -t mcp-server-gitparser .
+
+# Run the container
+docker run --rm -it -p 8000:8000 mcp-server-gitparser
+
+# Or use the helper script
+./scripts/start-docker.sh --rebuild
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)